test/e2e/epp/README.md

```diff
@@ -28,18 +28,28 @@ Follow these steps to run the end-to-end tests:
    export HF_TOKEN=<MY_HF_TOKEN>
```

Comment:
can you clarify that HF_TOKEN is required only if running the gpu based e2e test?
also in the pre-req section

Reply:
Commit [c5029cf](https://github.com/kubernetes-sigs/gateway-api-inference-extension/commit/c5029cf52e82a5e3dc3f1d317717afe386e57e4c) resolves this comment.